insights/specs/datasources/httpd.py

```diff
@@ -78,7 +78,7 @@ def _get_all_include_conf(root, glob_path):
                             section_number = section_number - 1
                         elif line.startswith("<"):
                             section_number = section_number + 1
-                        if section_number == 0 and (line.startswith("Include ") or line.startswith("IncludeOptional ")):
+                        elif section_number == 0 and (line.lower().startswith("include ") or line.lower().startswith("includeoptional ")):
```

Comment:
with the `line.lower()` being introduced, a tiny computing will be done before the two `startswith`, in this case, better change to one `startswith` as 
```
line.lower().startswith(("include ", "includeoptional "))
```